docs/en/stack/security/authentication/realms.asciidoc

```diff
@@ -108,6 +110,38 @@ realm type dictates its own set of required and optional settings. That said,
 there are 
 {ref}/security-settings.html#ref-realm-settings[settings that are common to all realms]. 
 
+[[authorization_realms]]
+==== Delegating authorization to another realm
+
+Some realms have the ability perform _authentication_ internally, but delegate the
+lookup and assignment of roles (that is, _authorization_) to another realm.
+
+For example, you may wish to use a PKI realm to authenticate your users with
+TLS client certificates, but then lookup that user in an LDAP realm, and use
+their LDAP group assignments to determine their roles in Elasticsearch.
+
+For realms that support this feature, it can be enabled by configuring the
+`authorization_realms` setting on the authenticating realm. Check the list of
+supported settings for each realm to see if they support the
+`authorization_realms` setting.
+
+Any realm that supports retrieving users (without needing their credentials)
+may be used as an _authorization realm_ (that is, its name may appear as one of
+the values in the list of `authorization_realms`). See <<run-as-privilege>> for
+further explanation on which realms support this.
+
+If delegated authorization is enabled for a realm, then it will authenticate
+the user in its standard manner (including relevant caching) and then look for
+that user in the configured list of authorization realms, trying each realm in
+order. The user is retrieved by principal - the user must have identical
```

Comment:
Re "trying each realm in order..." is that the order specified in the authorization_realms setting or the order specified in the realm chain?  I think the former